public_html/wp-content/plugins/wc-fonts/wc-fonts.php

```diff
@@ -177,6 +210,21 @@ public function field_font_awesome_url() {
 		<?php
 	}
 
+	/**
+	 * Settings API field for the Font Awesome kit ID
+	 */
+	public function field_font_awesome_kit() {
+		$value = isset( $this->options['font-awesome-kit'] ) ? $this->options['font-awesome-kit'] : '';
+		?>
+
+		<input type="text" name="wc-fonts-options[font-awesome-kit]" value="<?php echo esc_attr( $value ); ?>" class="regular-text code" />
+		<p class="description">
+			<?php esc_html_e( 'Enter your Font Awesome Kit ID only. Do not add any URLs or JavaScript.', 'wordcamporg' ); ?>
```

Comment:
Might be worth adding a line here, or on the other Font Awesome field (or both) explaining that you only need to use one field or the other.

Reply:
And maybe that the kit is only relevant for v5 and above.

Reply:
Technically you _can_ use both, v5 has some kind of version conflict magic ✨  (i mean, you still shouldn't.)

You're just suggesting adding some text, not doing any detection on whether one field is filled out or not, right?

Reply:
Yeah, just some text